src/ProgressOnderwijsUtils/Html/HtmlHelpers.cs

```diff
@@ -62,21 +62,21 @@ public static HtmlFragment EmptyIfNull<TContent>(this TContent? htmlFragmentOrNu
             where TContent : struct, IConvertibleToFragment
             => htmlFragmentOrNull?.AsFragment() ?? HtmlFragment.Empty;
 
-        public static HtmlFragment AsFragment([CanBeNull] this string textContent)
+        public static HtmlFragment AsFragment(this string? textContent)
             => string.IsNullOrEmpty(textContent) ? HtmlFragment.Empty : HtmlFragment.TextContent(textContent);
 
         public static HtmlFragment Append<T>([CanBeNull] this T head, HtmlFragment tail)
             where T : IConvertibleToFragment
             => (head?.AsFragment() ?? HtmlFragment.Empty).Append(tail);
 
-        public static HtmlFragment Append<T>([CanBeNull] this T head, params HtmlFragment[] longTail)
+        public static HtmlFragment Append<T>([CanBeNull] this T head, params HtmlFragment[]? longTail)

```

Comment:
You didn't remove CanBeNull here; oversight?

Reply:
I didn't proactively remove CanBeNull anywhere, R# removes it when you type '?'.

Reply:
oh... just.... sometimes only, apparently... :-D

Reply:
Not adding ? is the oversight.

Reply:
or not, you can't add ? to an unconstrained generic type.

Reply:
okidoki.